frontend/src/employee-frontend/components/reports/VoucherServiceProviderUnit.tsx

```diff
@@ -366,15 +383,36 @@ function VoucherServiceProviderUnit() {
                             : row.type
                         }
                       >
-                        <Link to={`/child-information/${row.childId}`}>
-                          {formatName(
-                            row.childFirstName,
-                            row.childLastName,
-                            i18n
-                          )}
-                        </Link>
-                        <br />
-                        {row.childDateOfBirth.format()}
+                        <FixedSpaceColumn spacing="xs">
+                          <Link to={`/child-information/${row.childId}`}>
+                            {formatName(
+                              row.childFirstName,
+                              row.childLastName,
+                              i18n
+                            )}
+                          </Link>
+
+                          <FixedSpaceRow spacing="s">
```

Comment:
The spacing is probably closer to `xs` in the spec, but otherwise everything looks great 😄